task.js

```diff
@@ -0,0 +1,30 @@
+function reverseListDrinks(){
+    let usersListElement = document.querySelector('.drinks-list');
+  
+    let listElements = [];
+  
+    let node = usersListElement.firstElementChild ;
+    while(node != null) { 
+      listElements.push(node.textContent);
+      node = node.nextElementSibling;
+    }; 
```

Comment:
I suspect you found this piece of code on the Internet. Why not, but
1. don't ever use non-strict equality/non-equality `!==`;
2. If you have a condition with equality (non-equality) of falsy values (false, null, undefined, '', 0, NaN) you can simplify it like `(node)`
3. this code should be simplified as
```suggestion
   	const listElements = Array.from(usersListElement.querySelectorAll('li'));

```